jupyter/Notebook Binding/Bar Tile INDEX.ipynb

```diff
@@ -0,0 +1,69 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table>\n",
+    "    <tr>\n",
+    "        <td><img src='../SystemLink_icon.png' /></td>\n",
+    "    </tr>\n",
+    "</table>\n",
+    "\n",
+    "## Bar Tile Binding Example\n",
+    "***\n",
+    "This example illustrates how a notebook execution can be bound to bar chart tile in a dashboard builder.\n",
+    "***\n",
+    "# Prerequisites\n",
+    "- None\n",
+    "\n",
+    "# Summary \n",
+    "This example illustrates how a notebook execution can be bound to bar tile in a dashboard builder. The result is set into the result object and then recorded using papermill to be consumed by the Notbook service. This example shows how a bar chart data can be modelled in the 'INDEX' format. In Index format, data is described as [y0, y1, y2, ...] format and x-axis labels are assumed to [0, 1, 2, ....] by default.\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import papermill as pm\n",
+    "\n",
+    "result = [];\n",
+    "result.append({\n",
+    "    'plot_style': 'BAR',\n",
```

Comment:
We should add titles to the charts with the `'title'` option. @BKnight760 in addition, do we also want to also show the graph orientation controls in these examples? If not, we could at least describe the option in markdown.